include/openPMD/IO/ADIOS/ADIOS2IOHandler.hpp

```diff
@@ -854,6 +861,11 @@ class ADIOS2IOHandler : public AbstractIOHandler
         return "ADIOS2";
     }
 
+    bool fullSupportForVariableBasedEncoding() const override
+    {
+        return openPMD_HAS_ADIOS_2_9;
```

Comment:
Maybe exchange for constant true once the minimum supported version of ADIOS2 has been raised to 2.9

Reply:
done :)